docs/developer-docs/latest/setup-deployment-guides/deployment/optional-software/snippets/strapi-server.md

```diff
@@ -0,0 +1,95 @@
+### Strapi Server
+
+In order to take full advantage of a proxied Strapi application you will need to configure Strapi to make it aware of the upstream proxy. Like with the below configurations there are 3 matching examples. To read more about the server and admin configuration files please see the [server configuration](/developer-docs/latest/setup-deployment-guides/configurations/required/server.md) and [admin configuration](/developer-docs/latest/setup-deployment-guides/configurations/required/admin-panel.md).
+
+:::note API Prefix
+Note that in many of these examples we are using the default API Prefix of `/api`, this can be changed without the need to directly modify the Nginx configuration, please see the [API Prefix](/developer-docs/latest/setup-deployment-guides/configurations/optional/api.md) documentation for more information.
```

Comment:
```suggestion
These examples use the default API Prefix of `/api`. This can be changed without the need to directly modify the Nginx configuration (see the [API prefix](/developer-docs/latest/setup-deployment-guides/configurations/optional/api.md) documentation).
```

to match our styleguide as well :-)